pepper-apis/src/test/java/org/broadinstitute/ddp/route/PutFormAnswersRouteStandalone2Test.java

```diff
@@ -0,0 +1,47 @@
+package org.broadinstitute.ddp.route;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+import java.time.Instant;
+
+import io.restassured.http.ContentType;
+import org.broadinstitute.ddp.constants.ErrorCodes;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.ddp.db.dao.JdbiRevision;
+import org.broadinstitute.ddp.db.dao.QuestionDao;
+import org.broadinstitute.ddp.db.dto.ActivityInstanceDto;
+import org.broadinstitute.ddp.model.activity.definition.QuestionBlockDef;
+import org.broadinstitute.ddp.model.activity.definition.validation.RequiredRuleDef;
+import org.broadinstitute.ddp.model.activity.revision.RevisionMetadata;
+import org.junit.Test;
+
+/**
+ * The test moved to a separate class because result of other tests affected on it and
+ * it's behaviour was incorrect
+ */
+public class PutFormAnswersRouteStandalone2Test extends PutFormAnswersRouteStandaloneTestAbstract {
```

Comment:
Do we need to add these new route tests to `ServerInSameProcessIntegrationTestSuite`?

Reply:
Yes, thanks. Fixed.